README.md

```diff
@@ -564,7 +564,63 @@ Chainweb.Pact.local(cmd, network_id: :testnet04, chain_id: 1)
    tx_id: nil
  }}
 ```
+### Poll endpoint
 
+Retrieves the result of a transaction or multiple transactions, you will need to include the request Keys.
+
+```elixir
+Kadena.Chainweb.Pact.poll(request_keys, network_opts \\ [network_id: :testnet04, chain_id: 0])
+```
+
+**Parameters**
+
+- `request_keys`: Unique ID of a pact transaction consisting of its hash, are the result of sending commands to the send endpoint.
```

Comment:
@EdwinGuayacan we need to specify the type of `request_keys`, it's pretty important. 

My suggestion:
- `request_keys`: List of strings. A request key is the unique id of a Pact transaction consisting of its hash, it is obtained from submitting a command via the `send` endpoint.

There is room for improvement, but not sure if the explanation of what a request key is looks good... @juanhurtado10 @L-Zuluaga